tests/components/unifi/test_sensor.py

```diff
@@ -110,9 +125,15 @@ async def test_sensors(hass):
     wireless_client_tx = hass.states.get("sensor.wireless_client_name_tx")
     assert wireless_client_tx.state == "6789.0"
 
+    wireless_client_uptime = hass.states.get("sensor.wireless_client_name_uptime")
+    assert wireless_client_uptime.state == "2020-09-15T14:41:00+00:00"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6184085)